pybliometrics/scopus/affiliation_search.py

```diff
@@ -33,10 +33,11 @@ def affiliations(self) -> Optional[List[NamedTuple]]:
             name = item.get('affiliation-name')
             variants = [d.get('$', "") for d in item.get('name-variant', [])
                         if d.get('$', "") != name]
+            parent = make_int_if_possible(item.get('parent-affiliation-id')) if item.get('parent-affiliation-id') != '0' else None
```

Comment:
Much easier:
``make_int_if_possible(item.get('parent-affiliation-id')) or None`
Since there is always a number, it will often become 0. 0 evaluates to null in an if-comparison, hence ``"0"` becomes `None`.

Reply:
Good point!